docs/en/stack/ml/df-analytics/ml-supervised-workflow.asciidoc

```diff
@@ -0,0 +1,142 @@
+next step is to produce a high-quality data set in {es} with a clear 
+relationship to your training objectives. If your data is not already in {es} or 
+you are not collecting all of the necessary information, this is the stage where 
+you develop your data pipeline. If you want to learn more about how to ingest 
+data into {es}, refer to the {ref}/ingest.html[documentation].
+
+{classification-cap} and {regression} are supervised {ml} techniques, therefore 
+you must 
+supply a labelled data set for training. This is often called the "ground truth" 
+as it consists of the observed examples on which you can train the model. The 
+"ground truth" allows the training process to identify relationships among the 
+various characteristics of the data and the predicted value as well as takes a 
+critical role in model evaluation.
+
+An important requirement is a data set that is large enough to train a model. 
+For example, if you would like to train a {classification} model that decides 
+whether an email is a spam or not, you need a labelled data set that contains 
+enough data points from each possible category to train the model. What counts 
+as "enough" depends on various factors like the complexity of the problem or 
+the {ml} solution you have chosen. There is no exact number that fits every 
+use case; deciding how much data is acceptable is rather a heuristic process 
+that might involve iterative trials.
+
+Before you train the model, consider preprocessing the data. In practice, the 
+type of preprocessing depends on the nature of the data set. Preprocessing can 
+include, but is not limited to, mitigating redundancy, reducing biases, applying 
+standards and/or conventions, data normalization, and so on.
+
+{regression-cap} and {classification} requires specifically structured source 
+data: a two dimensional tabular data structure. For this reason, you might need 
+to {ref}/transforms.html[{transform}] your data to create a {dataframe} which 
+can be used as the source for {dfanalytics} like {regression} and 
+{classification}.
+
+[[train-test-iterate]]
+== Train, test, iterate
+
+After your data is prepared and transformed into the right format, it is time to 
+train the model. Training is an iterative process — every round of iteration is 
+followed by an evaluation to see how the model performs. When you are satisfied 
+with the results, you are ready to deploy the model otherwise you may want to 
+adjust the training configuration or consider alternative ways to preprocess and 
+represent your data.
+
+The first step is defining the features – the relevant fields in the data set – 
+that will be used for training the model. By default, all the fields with 
+supported types are included in classification and regression automatically. 
+However, you can exclude irrelevant fields optionally from the process. Doing so 
+makes a large dataset more manageable and by reducing the computing resources 
+and time required for training.
```

Comment:
typo? `Doing so makes a large dataset more manageable, reducing the computing resources 
and time required for training.`

Reply:
Yes, I think we edited out something incompletely. Thanks for spotting it!